docs/database/mirrored-database/azure-sql-database-troubleshoot.md

```diff
@@ -25,6 +25,7 @@ For troubleshooting the automatically configured mirroring for Fabric SQL databa
 | Workspace deleted | Mirroring stops automatically | If mirroring is still active on the Azure SQL Database, execute the following stored procedure on your Azure SQL Database: `exec sp_change_feed_disable_db;`. |
 | Fabric trial capacity expired |  Mirroring stops automatically | See [Fabric trial capacity expires](../../get-started/fabric-trial.md#the-trial-expires). |
 | Fabric capacity exceeded | Mirroring will pause | Wait until the overload state is over or update your capacity. Learn more from [Actions you can take to recover from overload situations](../../enterprise/throttling.md#actions-you-can-take-to-recover-from-overload-situations). Mirroring will continue once the capacity is recovered. |
+| Any other terminal errors |  Mirroring will be disabled | To ensure your compute resources are not impacted and to minimize impact on operational Azure SQL database, Mirroring will be disabled on any persistent/terminal errors |
```

Comment:
@im-microsoft can we elaborate what are "persistent/terminal" errors, and how to check if SQL publisher is disabled?

Reply:
@linda33wj, sure, some of the persistent/terminal errors are Fabric Workspaces got deleted with SQL Publishing still running, SQL Publishing unable to write to LZ due to capacity paused/exceeded etc. Way to check if SQL publisher is disabled is to execute this SQL query: 

SELECT is_data_lake_replication_enabled from sys.databases

If returned value is 0, it indicates SQL publisher is disabled.

Also, users can run the public stored procedure below, which is documented in the troubleshooting section below:

EXEC sp_help_change_feed;
The key columns to look for here are the table_name and state. Any value besides 4 indicates a potential problem.

Reply:
Customers don't know what a terminal error is. Can we say "persistent resource errors"? @im-microsoft

Reply:
sure @WilliamDAssafMSFT, i was contemplating on the terminal error verbiage. "persistent resource errors" or "persistent errors" both work